guides/common/attributes-satellite.adoc

```diff
@@ -107,5 +107,5 @@
 :RepoRHEL7ServerSatelliteToolsProductVersion: rhel-7-server-satellite-client-7-rpms
 :RepoRHEL8ServerSatelliteToolsProductVersion: satellite-client-7-for-rhel-8-<arch>-rpms
```

Comment:
while not exactly related to the below change, I think this should be adjusted too!
```suggestion
:RepoRHEL7ServerSatelliteToolsProductVersion: rhel-7-server-satellite-client-6-rpms
:RepoRHEL8ServerSatelliteToolsProductVersion: satellite-client-6-for-rhel-8-<arch>-rpms
```

CC @ehelms to double check the repo name

Reply:
Ok. I am considering it 👍